packages/api/src/controllers/webhooksController.ts

```diff
@@ -115,6 +126,59 @@ export default async function webhookController(fastify: FastifyInstance) {
     },
   );
 
+  fastify.withTypeProvider<TypeBoxTypeProvider>().post(
+    "/amazonses",
+    {
+      schema: {
+        description: "Used to consume amazonses notification events.",
+        tags: ["Webhooks"],
+        body: AmazonSNSEvent,
+      },
+      onRequest: (req, _, done) => {
```

Comment:
Why is this necessary?

Reply:
Amazon sends webhooks with a JSON body and a content type of "text/plain", annoyingly.  We could just JSON.parse the body in the handler, but this way the schema checking can still do its thing in fastify.

Reply:
That's annoying! Can you leave a short comment to that effect?